### ISSUE TITLE
[crash] using console mode 

### ISSUE THREAD

**Issue by LasneF** [pb33f/openapi-changes#115]
leveraging the latest version got this stack 

Notice running vaccum on each file  does not crash ... 

```
documentations\datamart-reports> D:\tmp\openapi-changes.exe console .\component-descriptor.yml .\component-descriptor2.yml

@@@@@@@   @@@@@@@   @@@@@@   @@@@@@   @@@@@@@@
@@@@@@@@  @@@@@@@@  @@@@@@@  @@@@@@@  @@@@@@@@
@@!  @@@  @@!  @@@      @@@      @@@  @@!
!@!  @!@  !@   @!@      @!@      @!@  !@!
@!@@!@!   @!@!@!@   @!@!!@   @!@!!@   @!!!:!
!!@!!!    !!!@!!!!  !!@!@!   !!@!@!   !!!!!:
!!:       !!:  !!!      !!:      !!:  !!:
:!:       :!:  !:!      :!:      :!:  :!:
 ::        :: ::::  :: ::::  :: ::::   ::
 :        :: : ::    : : :    : : :    :


openapi-changes version: 0.0.55 | compiled: Mon, 26 Feb 2024 21:16:51 UTC

  DONE   starting console...


panic: reflect: call of reflect.Value.Elem on slice Value

goroutine 1 [running]:
reflect.Value.Elem({0xd755c0?, 0xc001e870a8?, 0x0?})
        C:/hostedtoolcache/windows/go/1.22.0/x64/src/reflect/value.go:1277 +0x195
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc002010d00, {0xdfb860, 0xc001e870a0})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:59 +0x40af
github.com/pb33f/openapi-changes/tui.DigIntoObject[...](0xc002010c80, 0xc002010d00, {0xdfb860, 0xc001c35188?, 0xdfb860?})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:311 +0xf6
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc002010c80, {0xdfb960, 0xc001c35180})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:144 +0x326f
github.com/pb33f/openapi-changes/tui.DigIntoObject[...](0xc002010b00, 0xc002010c80, {0xdfb960, 0xc002154300?, 0xdfb960?})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:311 +0x127
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc002010b00, {0xdfb960, 0xc0021542a0})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:109 +0x389f
github.com/pb33f/openapi-changes/tui.BuildTreeMapNode({0xdd3320?, 0xc001c35028?, 0x0?}, 0xc002010a80)
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:294 +0x1fe
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc002010880, {0xdfb960, 0xc001c34fc0})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:171 +0x2ecc
github.com/pb33f/openapi-changes/tui.DigIntoObject[...](0xc002010800, 0xc002010880, {0xdfb960, 0xc000dbb268?, 0xdfb960?})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:311 +0x127
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc002010800, {0xdfbbe0, 0xc000dbb260})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:109 +0x389f
github.com/pb33f/openapi-changes/tui.BuildTreeMapNode({0xdd3200?, 0xc000f22c28?, 0x0?}, 0xc002010780)
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:294 +0x1fe
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc002010580, {0xdfbf60, 0xc000f22c00})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:213 +0x250c
github.com/pb33f/openapi-changes/tui.BuildTreeMapNode({0xdd32c0?, 0xc001e87048?, 0x0?}, 0xc0022bf300)
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:294 +0x1fe
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc0022bf300, {0xdfbfe0, 0xc001e87040})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:164 +0x2f45
github.com/pb33f/openapi-changes/tui.DigIntoObject[...](0xc0003cbb00, 0xc0022bf300, {0xdfbfe0, 0xc0003cbba8?, 0xdfbfe0?})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:311 +0x113
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc0003cbb00, {0xdfc1e0, 0xc0003cbb80})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:149 +0x31b9
github.com/pb33f/openapi-changes/tui.DigIntoObject[...](0xc0003cb680, 0xc0003cbb00, {0xdfc1e0, 0xc000a20008?, 0x0?})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:311 +0x127
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc0003cb680, {0xdfc260, 0xc000a20000})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:84 +0x3cdf
github.com/pb33f/openapi-changes/tui.BuildTreeMapNode({0xdd3260?, 0xc001692968?, 0x0?}, 0xc0003cb580)
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:294 +0x1fe
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc0003cb080, {0xdfc860, 0xc001692960})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:232 +0x214d
github.com/pb33f/openapi-changes/tui.DigIntoObject[...](0xc0003cab00, 0xc0003cb080, {0xdfc860, 0xc0007de010?, 0xdfc860?})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:311 +0xff
github.com/pb33f/openapi-changes/tui.buildTreeNode(0xc0003cab00, {0xdfc660, 0xc0007de000})
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:80 +0x3d93
github.com/pb33f/openapi-changes/tui.BuildTreeModel({0x12e2568?, 0xc00087e3c0?}, 0xc0007de000)
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:35 +0xbf
github.com/pb33f/openapi-changes/tui.BuildTreeView(0xc000598750?)
        D:/a/openapi-changes/openapi-changes/tui/build_tree.go:20 +0x3a
github.com/pb33f/openapi-changes/tui.BuildApplication({0xc000e181c0, 0x2, 0x2}, {0x12ceb84, 0x6})
        D:/a/openapi-changes/openapi-changes/tui/main_console.go:24 +0x66
github.com/pb33f/openapi-changes/cmd.runLeftRightCompare({0xc000024240, 0x1a}, {0xc000024280, 0x1b}, 0xc00029e060, 0xc00029e0c0, {0x0, 0x0}, 0x1)
        D:/a/openapi-changes/openapi-changes/cmd/console.go:382 +0x810
github.com/pb33f/openapi-changes/cmd.init.0.GetConsoleCommand.func1(0xc00014c308, {0xc000292240, 0x2, 0xf2ffd0?})
        D:/a/openapi-changes/openapi-changes/cmd/console.go:248 +0x9af
github.com/spf13/cobra.(*Command).execute(0xc00014c308, {0xc0002921e0, 0x2, 0x2})
        C:/Users/runneradmin/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:983 +0xaca
github.com/spf13/cobra.(*Command).ExecuteC(0x194e380)
        C:/Users/runneradmin/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
        C:/Users/runneradmin/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/pb33f/openapi-changes/cmd.Execute({0x12ceb84?, 0x19?}, {0x1
```

**Comment by daveshanley** [comment]
Will fix.

**Comment by daveshanley** [comment]
Fixed in `v0.0.56` Added test to verify.
